security/pkg/util/jwtutil.go

```diff
@@ -90,24 +90,35 @@ type jwtPayload struct {
 // Clients should not use unbound tokens except in cases where
 // bound tokens are not possible.
 func IsK8SUnbound(jwt string) bool {
+	aud, f := ExtractJwtAud(jwt)
+	if !f {
+		return false // unbound tokens are valid JWT
```

Comment:
Based on the function comments of IsK8SUnbound() that a K8S unbound token is a JWT with no audience, when the aud is failed to be extracted from the JWT, "return false" should be "return true" instead because of no aud?

Reply:
My understanding: there is a difference between "we parsed the JWT and it had no audiences" and "The token was opaque and couldn't be parsed as a jwt, so we couldn't read the audiences"

However, I am also keeping existing behavior here